pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -209,3 +212,11 @@ func (c *contract) callTx(ctx context.Context, callData []byte) ([]byte, error)
 	}
 	return result, nil
 }
+
+func (c *contract) setFee(in *big.Int) {
+	c.accumulativeFee = in
+}
+
+func (c *contract) GetFee() *big.Int {
```

Comment:
The naming convention for Golang getters is that they do not contain the `Get` prefix.